docs/infrastructure/deployment-targets/linux/ssh-targets/requirements.md

```diff
@@ -0,0 +1,101 @@
+---
+title: SSH Target Requirements
+description: Requirements for using SSH deployment targets with Octopus.
+position: 5
+---
+
+SSH deployment targets must meet the following requirements:
+
+- It is accessible through SSH and SFTP (See [creating an SSH Key Pair](/docs/infrastructure/accounts/ssh-key-pair.md#Creating-a-SSH-Key-Pair)).
+- The `$HOME` environment variable must be available.
+- `bash` 3+ is available at `/bin/bash`. (It does not need to be the user’s default shell.)
+- `tar` is available. This is used to unpack Calamari.
+- `base64` is available. This is used for encoding and decoding variables.
+- `grep` is available.
+
+## Bash Startup Files
+
+When connecting to a target over SSH, the Octopus server connects then executes the script via the `/bin/bash` command to ensure it is running with a bash shell (and not the default terminal shell for that user). Any login scripts that you wish to run should therefore be put into the `.bashrc` script file since this is invoked for non-login shells.
+
+For example, with targets on a Mac the default $PATH variable may be missing `/usr/sbin`. This can be added in the `.bashrc` script with the line:
+
+> PATH=$PATH:/usr/sbin
+
+If the `.bashrc` file doesn't already exist, create it in the user folder of the user that is connecting to the Max OSX instance. If the remote user is called `octopus`, then this file will be located at `/Users/octopus/.bashrc`.
+
+See the Bash Reference Manual, section [6.2 Bash Startup Files](http://www.gnu.org/software/bash/manual/bashref.html#Bash-Startup-Files) for more information about startup scripts.
+
+## .NET {#dotnet}
+
+[Calamari](/docs/api-and-integration/calamari.md) is the command-line tool that is invoked to perform the deployment steps on the deployment target. Calamari runs on .NET.  SSH targets can be configured to use either: 
+
+- [Self-contained Calamari](#self-contained-calamari) built against .NET Core.
+- [Calamari on Mono](#mono-calamari) built against the full .NET framework.
+
+### Self-contained Calamari {#self-contained-calamari}
```

Comment:
Super minor point. It. might be good to make this a level 2 (##) heading and then it will be the same as Calamari on Mono, and the level 4 headings can become level 3. The recent style changes make level four headings look like normal text.

Reply:
Done.